packages/react-devtools-shared/src/inspectedElementCache.js

```diff
@@ -63,6 +63,19 @@ function getRecordMap(): WeakMap<Element, Record<InspectedElementFrontend>> {
   return getCacheForType(createMap);
 }
 
+/**
+ * Removes inspected element metadata from the cache.
+ *
+ * This function should only be called during render (since it reads from Context)
+ * and the callback it returns should be called prior to render.
+ */
+export function getClearElementCallback(element: Element): () => void {
+  const map = getRecordMap();
+  return function clearElementCallback(): void {
+    map.delete(element);
+  };
+}
```

Comment:
This is the part I'm asking for feedback about: What is our recommended pattern for clearing (either an entry or entirely) the Suspense cache before retrying a render? Feels a little awkward to do it this way. Maybe there's a better/easier way?

Reply:
Maybe I should use the `refresh` API for this instead, and just pass an `undefined` value.